src/client/components/dragAndDrop/dragAndDrop.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import Dropzone from 'react-dropzone';
+import { createAudioData, playPauseSoundFromFile } from '../../actions/audioActions';
+
+import './dragAndDrop.css';
+
+
+class DragAndDrop extends Component {
+    onDrop = (acceptedFiles) => {
+      const file = acceptedFiles[0];
```

Comment:
You could use destruction here const [file] = acceptedFiles;

Reply:
done